src/index.js

```diff
@@ -8,6 +8,7 @@ angular
         $httpBackend.whenGET('/api/threads').respond(mockBackend.getThreads);
         $httpBackend.whenGET('/api/updates').respond(mockBackend.getUpdates);
         $httpBackend.whenGET('/api/owner').respond(mockBackend.getOwner);
+        $httpBackend.whenGET('/api/screenTypes').respond(mockBackend.getScreenTypes);
```

Comment:
Because I think screen types is defined in mock-backend.js. We may define it in card-service or new screen service as a enum.